backend/controllers/feedback.controller.js

```diff
@@ -98,10 +91,8 @@ const deleteFeedback = async(req, res) => {
             })
         }
         res.status(200).json({
-            results_length: feedback.length,
-            data: {
-                feedbackdata: feedback
-            }
+            status: "success",
+            feedbackdata: feedback
```

Comment:
```suggestion
            data: feedback
```

Should delete be returning the feedback?

Reply:
not necessary, but standardised it to follow w other apis that return the deleted resource (e.g. delete user)